index.yaml

```diff
@@ -21,14 +21,14 @@ entries:
     - https://github.com/rh-ecosystem-edge/console-plugin-nvidia-gpu
     type: application
     urls:
-    - console-plugin-nvidia-gpu-0.1.0.tgz
-    version: 0.1.0
+    - console-plugin-nvidia-gpu-0.2.4.tgz
+    version: 0.2.4
   - apiVersion: v2
     appVersion: latest
```

Comment:
now we have two versions with `appVersion: latest`. Can you change this one to `0.2.3` ?

Reply:
Thank you for the response on this PR!

This file is automatically generated by running `helm index .`.  The value of `appVersion: latest` is metadata that is extracted from the Chart.yaml file in the .tgz.

If I manually update that value, it may fail hash checks when a user attempts to install that version of the chart (I haven't dug into this to test, but I don't think this is a good idea).

The correct way to fix this is to go back to the source the chart file that generated the 0.2.3 version and update it there, then repackage that version of the chart.  The other option would be to extract the .tgz for 0.2.3 here, edit it then package a new version of 0.2.3.  With either of those options, though you may cause some issues from anyone pinning to that version of the chart since the hash is once again changing for a released version.  Generally you don't want to edit released chart versions.

With that being said, it is perfectly acceptable to have multiple charts using the same appVersion.  Using `latest` as the appVersion is usually an indication that both of the versions of the chart should work with the latest image tag.

I'm happy to update it if you feel strongly but this is not something I would advise.

Reply:
@strangiato thanks for the explanation. I totally forgot this file is actually generated by helm. LGTM!